astro/src/content/blog/llms-for-docs.mdx

```diff
@@ -0,0 +1,231 @@
+* `llms-full.txt` is a similar list but includes the full text rather than just the URL of the pages. Here is our [docs `llms-full.txt` file](/docs/llms-full.txt). 
+
+It's unclear if you can have more than one set of `llms.txt` files for different sections of your site; the standard is still evolving. [Here's an open issue to discuss this](https://github.com/AnswerDotAI/llms-txt/issues/59).
+
+These files are typically located at the root of your domain: `https://example.com/llms.txt` for example. This is so LLMs have one place to look.
+
+If you put these files somewhere else or have more than one of them, add a link from your HTML pages to the files. Crawlers for LLMs hunt around, but not too much. We shipped an `llms.txt` file for our docs in December, but it wasn't at the root location above. This file was downloaded by the OpenAI chatbot nearly 60 times as much after we linked to it from our HTML docs pages as before we did.
+
+In addition, when you make these files available to developers browsing your site, they can retrieve it and upload it to an LLM of their choosing. This allows them to interrogate the docs or learn more about the doc structure using their tool of choice.
+
+Ensure these files are automatically updated in the same way your `sitemap.xml` file is. You want any new or updated documentation to be included with minimal effort.
+
+Here's a table showing the relative percentages of downloads of either our `llms.txt` file or our `llms-full.txt` file, Jan 1 to Jun 5. There may be other LLMs downloading this file as well.
+
+| Scraper                     | Percent Of LLMs Downloading |
+|-----------------------------|--------------------|
+| OpenAI                      | 71.59 | 
+| GoogleBot (used for Gemini) | 17.20 | 
+| Perplexity                  | 8.22 |
+| BingBog (used for Copilot)  | 2.80 | 
+| Anthropic                   | 0.19 |
+
+### Implementation Notes
+
+We build these files for our docs section. Here's the code for the `llms.txt` file:
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-site/refs/heads/main/astro/src/pages/docs/llms.txt.ts" lang="javascript" />
+
+You could make this more sophisticated by giving documents in the collection rankings, but we haven't done that yet. For other sections of our website, we handcrafted this file.
+
+For our documentation `llms-full.txt`, the code looks similar: 
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-site/refs/heads/main/astro/src/pages/docs/llms-full.txt.ts" lang="javascript" />
+
+This pulls in the entire body (`${doc.body}`) rather than just the URL, but other than that is the same.
+
+We use a lot of includes in our docs. I did some work to inline the include file markdown, but that ended up making the `llms-full.txt` file too big for the popular LLMs' context windows.
+
+### Success Metrics
+
+Are these files downloaded? How often? By what user agents? Is it being primarily downloaded by LLMs or users?
```

Comment:
"these files" referred to as "it" in the following sentence. Maintain consistent plurality.